Assets/Mirror/Components/NetworkAnimator.cs

```diff
@@ -152,7 +152,7 @@ void SendAnimationMessage(int stateHash, float normalizedTime, int layerId, byte
             {
                 RpcOnAnimationClientMessage(stateHash, normalizedTime, layerId, parameters);
             }
-            else if (ClientScene.readyConnection != null)
+            else if (client.connection != null)
```

Comment:
This is so much better!

We could even merge this  without the rest of the stuff.